.github/workflows/publish-qemu-bin.yml

```diff
@@ -0,0 +1,163 @@
+name: Publish Qemu External Dependency
+
+on:
+  workflow_dispatch:
+  release:
+    types: [published]
+  pull_request:
+    branches:
+      - main
+    paths:
+      - .github/workflows/publish-qemu-bin.yml
+      - .github/publish-qemu-config.yml
+
+env:
+  QEMU_VERSION_FILE: .github/publish-qemu-config.yml
+
+jobs:
+  qemu-windows:
+    name: Build Qemu for Windows
+    runs-on: windows-latest
+  
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v4
+    - name: Get current Qemu version
+      id: qemu-version
+      run: |
+        $qemuVersion = (Get-Content -Path $env:QEMU_VERSION_FILE | ConvertFrom-Json).windows
+        "QEMU_VERSION=$qemuVersion" | Out-File -FilePath "$env:GITHUB_OUTPUT" -Append
+      shell: pwsh
+    - name: Download Qemu v${{ steps.qemu-version.outputs.QEMU_VERSION }}
+      env:
+        QEMU_VERSION: ${{ steps.qemu-version.outputs.QEMU_VERSION }}
+      run: |
+        choco install qemu --version=$env:QEMU_VERSION
+    - name: Stage Qemu Binaries
+      run: |
+        mkdir temp

```

Comment:
Not for now, but we should eventually consider building this ourselves. It's a huge pain to build QEMU for windows, but might be useful as the existing one has some issues with things like WHPX. Not for this PR, but a note for the future.

Reply:
I will mark this down as a TODO in a future PR :)